### DISCUSSION TITLE
24/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#191] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer estuve un ratito en la clase de PP para escuchar sobre lo que se pretende si final de la presentación. Tuve clase de tecnología solo media hs duro. 
Hoy quiero terminar las pantallas y ponerme con los trabajos de tecnología y emprendurismo.
Siempre... El tiempo...

**Comment by rpgrca** [comment]
Ayer se me pasó el día volando, tanto que hasta que me acordé de hacer la evidencia 3 a medianoche. De código no hice nada. Este fin de semana estoy con el trabajo para entregar de tecnología y desarrollo, tampoco voy a tocar código.
